cmd/create/cluster/cmd.go

```diff
@@ -508,6 +511,14 @@ func init() {
 			"roles in STS clusters.",
 	)
 
+	// Options releated to HyperShift:
+	flags.BoolVar(
+		&args.hostedClusterEnabled,
+		"hosted-cp",
+		false,
+		"Enable or disable use of Hosted Cluster mode.",
+	)
```

Comment:
Let's add a `flags.MarkHidden("hosted-cp")` here to not exposed non-supported features.